Dockerfile

```
@@ -0,0 +1,40 @@
+# ENV UV_SYSTEM_PYTHON=1
+# ENV UV_PROJECT_ENVIRONMENT=/app 
+
+COPY lib-new lib-new
+RUN uv --directory=lib-new export --format requirements.txt --no-editable --frozen | sed -e 's@^\.@# Elided: .@' /tmp/lib-new-constraints.txt 
+RUN uv --directory=lib-new build -b /tmp/lib-new-constraints.txt -o /tmp/dist --wheel
+
+COPY app-new app-new
+RUN uv --directory=app-new export --format requirements.txt --no-editable --frozen | sed -e 's@^\.@# Elided: .@' /tmp/app-new-constraints.txt 
+RUN uv --directory=app-new build -b /tmp/app-new-constraints.txt -o /tmp/dist --wheel
+
+RUN pip install --no-cache-dir --prefix=python-packages /tmp/dist/*.whl
```

Hadolint (2.12.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)